apps/etterlatte-behandling/src/main/kotlin/behandling/revurdering/RevurderingService.kt

```diff
@@ -61,18 +65,29 @@ class RealRevurderingService(
         sakId: Long,
         forrigeBehandling: Behandling,
         revurderingAarsak: RevurderingAarsak,
+        bruker: Bruker,
         kilde: Vedtaksloesning
     ): Revurdering {
         if (featureToggleService.isEnabled(RevurderingServiceFeatureToggle.OpprettManuellRevurdering, false)) {
             return inTransaction {
-                opprettRevurdering(
+                val revurdering = opprettRevurdering(
                     sakId,
                     forrigeBehandling,
                     revurderingAarsak,
                     null,
                     Prosesstype.MANUELL,
                     kilde
                 )
+
+                runBlocking {
```

Comment:
For automatisk regulering er eg ganske sikker på at dette blir gjort også i vilkaarsvurdering-kafka-steget, så må passe på at det ikkje blir feil pga dobbelt opp der.

Reply:
Er ganske sikker på at det kallet er idempotent idag, men uansett så har jeg etter noen diskusjoner med lars erik tenkt til å flytte denne `kopierFraForrige`-logikken til vilkårsvurderings-appen istedet.

D.v.s hvis behandling forholder seg kun til `vilkaarsvurderingService.opprett()` og vilkårsvurderingsappen for sørge for å kopiere forrige vilkårsvurderingen for revurderinger.